api/services/roundService.js

```diff
@@ -0,0 +1,32 @@
+
+export default class RoundService {
+    constructor(players) {
+        this.players = players;
+
+    }
+
+    newRound() {
+        getCzar(newGame)//TODO: newgame is a boolean that is passed in
+
+    }
+
+    endRound() {
+
+    }
+
+    getCzar(newGame) {
+        if (newGame) {
+            this.currentCzarNum = Math.floor(Math.random() * Math.floor(this.players.length));
+            this.currentCzar = this.players[this.currentCzarNum - 1];
+            return this.currentCzar;
```

Comment:
lines 20 and 21 also look similar to the duplicate code we put out before. I wonder if all the ifs together can just be used to work out `this.currentCzarNum` and then the last two lines of the function would just be what lines 28-29 are now